examples/basic-typescript/tsconfig.json

```diff
@@ -13,7 +13,14 @@
     "resolveJsonModule": true,
     "isolatedModules": true,
     "jsx": "preserve",
-    "incremental": true
+    "incremental": true,
+    "baseUrl": ".",
+    "paths": {
+      "@/components/*": ["components/*"],
+      "@/generated/*": ["generated/*"],
```

Comment:
We don't seem to have a `generated` folder in this repo. I'm guessing this was copy/pasted from another repo?

Reply:
Will update.

Reply:
Actually this is where icons are generated from the included `icons` command.

Reply:
Ah, got it, I missed that. Is that `icons` script being used in this example?

Reply:
We don't seem to have any icons under `public/icons` so maybe we just want to clean that stuff up?